packages/@tinacms/datalayer/src/node.ts

```diff
@@ -0,0 +1,154 @@
+import type { IncomingMessage, ServerResponse } from 'http'
+
+type NodeApiHandler = (
+  req: IncomingMessage,
+  res: ServerResponse
+) => Promise<void>
+
+type DatabaseClient = any
+
+export interface BackendAuthentication {
+  isAuthorized: (
+    req: IncomingMessage,
+    res: ServerResponse
+  ) => Promise<
+    | {
+        isAuthorized: true
+      }
+    | {
+        isAuthorized: false
+        errorMessage: string
+        errorCode: number
+      }
+  >
+  extraRoutes?: {
+    [key: string]: {
+      isAuthRequired?: boolean
```

Comment:
```suggestion
      protected?: boolean
```
or `secure`

Reply:
@kldavis4 `protected` is a reserved word when using strict mode. [see this web page for more info](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#future-proofing_javascript). I think it makes more sense to go with `isProtected` or `secure`

Reply:
updated to use `secure`